src/test/java/pages/LoginPage.java

```diff
@@ -39,6 +48,9 @@ public void providePassword(String password) {
 
     public void logIn(){
         provideEmail("terrence.regis@gmail.com");
+
```

Comment:
The code is not compiled! Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/215410677-18c266be-7b9a-4028-b401-26a47e73f9dd.png)

For example

```
package pages;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.support.FindBy;

public class LoginPage extends BasePage {

    //locators
    @FindBy(css = "[type='submit']")
    private WebElement submitButtonLocator;
    @FindBy(css = "[type='email']")
    private WebElement emailField;
    @FindBy(css = "[type='password']")
    private WebElement passwordField;


    public LoginPage(WebDriver givenDriver) {
        super(givenDriver);
    }


    public void clickSubmitBtn() {
        submitButtonLocator.click();
    }

    public void provideEmail(String email) {
        emailField.sendKeys(email);
    }

    public void providePassword(String password) {
        passwordField.sendKeys(password);

    }

    public void logIn() {
        provideEmail("terrence.regis@gmail.com");
        providePassword("te$t$tudent");
        clickSubmitBtn();

    }

}
```